myuw/static/vue/components/_templates/boilerplate/messages.vue

```diff
@@ -19,7 +19,8 @@
         <b-link v-b-modal.tourModal class="text-white">
           <u>See MyUW at a glance</u>
         </b-link>
-        <b-button-close text-variant="light" @click="hideOnboardMessage" />
+        <button type="button" class="btn-close" aria-label="Close"
```

Comment:
Char, this doesn't work. Any suggestion?

Reply:
This doesn't 1 to 1 translate. This provided by bootstrap-vue not base bootstrap.
It translates to
```html
<button type="button" aria-label="Close" class="close">×</button>
```
It also gets a event attached to it that I have not figured out yet. Best to leave this change out of this pr.